mysql-test/main/func_json.test

```diff
@@ -718,3 +718,103 @@ DROP TABLE t1;
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
```

Comment:
@mariadb-RuchaDeodhar , the end of tests should be below the added test case for ease of merging.